google-cloud-dataproc/synth.metadata

```diff
@@ -45,265 +45,5 @@
         "config": "google/cloud/dataproc/artman_dataproc_v1beta2.yaml"
       }
     }
-  ],
-  "newFiles": [
```

Comment:
@dazuma Just FYI, I noticed that `newFiles` is now being removed from the dataproc `synth.metadata`.